rust/operator-binary/src/controller.rs

```diff
@@ -919,6 +923,7 @@ fn build_rolegroup_statefulset(
             "config",
             "log",
             merged_config.logging.containers.get(&Container::Vector),
+            ResourceRequirements::default(),
```

Comment:
We must pass in dedicated resources constructed using `ResourceRequirementsBuilder`.